src/libraries/Microsoft.Extensions.Configuration.EnvironmentVariables/src/EnvironmentVariablesConfigurationProvider.cs

```diff
@@ -38,6 +38,13 @@ public class EnvironmentVariablesConfigurationProvider : ConfigurationProvider
         public override void Load() =>
             Load(Environment.GetEnvironmentVariables());
 
+        /// <summary>
+        /// Generates a string representing this provider name and relevant details.
+        /// </summary>
+        /// <returns> The configuration name. </returns>
+        public override string ToString()
+            => $"{GetType().Name} Prefix: '{_prefix}'";
```

Comment:
Should we special case if the prefix is empty and dump something like `<empty>` or you think `''` is fine?

Reply:
I was playing with it and I think '' is fine but I could do either.

Reply:
Don't have a strong opinion either, so whatever you think is fine, I'm fine, just throwing an idea.